utils/config.go

```diff
@@ -284,7 +284,7 @@ func EnsureConfigFile(fileName string) (string, error) {
 // LoadConfig will try to search around for the corresponding config file.  It will search
 // /tmp/fileName then attempt ./config/fileName, then ../config/fileName and last it will look at
 // fileName.
-func LoadConfig(fileName string) (config *model.Config, configPath string, appErr *model.AppError) {
+func LoadConfig(fileName string, isTimezoneConfig bool) (config *model.Config, configPath string, appErr *model.AppError) {
```

Comment:
Instead of requiring `LoadConfig(fileName, false)` everywhere, would it make sense to instead expose a `LoadTimezoneConfig`, and instead factor out a `loadConfigFile` for the common code?

Reply:
Sure, it wasn't that drastic of a refactor so I decided to add a bool instead.

not sure what your thoughts are on that. Only common code would be:

```
if fileName != filepath.Base(fileName) {
 		configPath = fileName
 	} else {
 		if path, err := EnsureConfigFile(fileName); err != nil {
 			appErr = model.NewAppError("LoadConfig", "utils.config.load_config.opening.panic", map[string]interface{}{"Filename": fileName, "Error": err.Error()}, "", 0)
 			return
 		} else {
 			configPath = path
 		}
 	}
 
 	config, err := ReadConfigFile(configPath, true)
 	if err != nil {
 		appErr = model.NewAppError("LoadConfig", "utils.config.load_config.decoding.panic", map[string]interface{}{"Filename": fileName, "Error": err.Error()}, "", 0)
 		return
 	}
```

Yeh, I can do that refactor